### PR TITLE
Get artifact/habitat starting tests on both Windows and Linux

### PR DESCRIPTION
**Pull request by miah**
This updates our artifact test and habitat plans to validate our artifact.

The tests are currently failing (this is great!)

https://buildkite.com/chef/inspec-inspec-master-artifact-habitat/builds/60#268643f9-9e5c-4893-9a4b-b29770b338df

Its great because they are executing at all. Now we iterate on useful tests and fixing the issues we discover.

### CODE REVIEW

File: .expeditor/buildkite/artifact.habitat.test.ps1 [hunk 1]
```diff
@@ -12,6 +12,15 @@ $Properties = 'Caption', 'CSName', 'Version', 'BuildType', 'OSArchitecture'
 Get-CimInstance Win32_OperatingSystem | Select-Object $Properties | Format-Table -AutoSize
 
 Write-Host "--- Installing the version of Habitat required"
+$hab_version = (hab --version)
+$hab_minor_version = $hab_version.split('.')[1]
+if ( -not $? -Or $hab_minor_version -lt 85 ) {
+    Write-Host "I'm being built with $hab_version. I need at least Hab 0.85.0, because I use the -IsPath option for setting/pushing paths in SetupEnvironment."
+throw "unable to build: required minimum version of Habitat not installed"
+} else {
+    Write-Host ":habicat: I think I have the version I need to build."
```
Comment (zenspider) [inline .expeditor/buildkite/artifact.habitat.test.ps1:20]: typo or emoji?

**Review by zenspider — APPROVED** [review]

File: .expeditor/buildkite/artifact.habitat.test.ps1 (hunk 1 above)
Comment (miah) [reply .expeditor/buildkite/artifact.habitat.test.ps1:20]: might be a emoji =)

**Comment by miah** [comment]
Squashed. This PR is ready for merge. Tests are still failing, but we are in a 100% better situation after this.

**Review by clintoncwolfe — APPROVED** [review]
Thanks - I'm glad it is failing in the way you want :-)

**Pull request merged**